lib/pages/faq/faqPage.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:acs_upb_mobile/pages/faq/model/question.dart';
+import 'package:acs_upb_mobile/pages/faq/service/question_service.dart';
+import 'QuestionsList.dart';
+import 'SearchWidget.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class FaqPage extends StatefulWidget {
+  @override
+  _FaqPageState createState() => _FaqPageState();
+}
+
+class _FaqPageState extends State<FaqPage> {
+  List<Question> questions = List<Question>();
+  List<String> categories;
+  String filter = "";
+  bool searchClosed = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return AppScaffold(
+      title: 'FAQ',
+      actions: [
+        AppScaffoldAction(
+          icon: Icons.search,
+          onPressed: () {
+            setState(() {
+              searchClosed = !searchClosed;
+            });
+          },
+        )
+      ],
+      body: ListView(
+        children: [
+          SearchWidget(
+            title: Text(
+              'Top Questions',
```

Comment:
Never hard-code strings. These should be localized - add the string to both `arb` files in the `l10n` folder (e.g. "labelTopQuestions") and fetch it here using `S.of(context).labelTopQuestions`. (Make sure you have the `Flutter Intl` plugin installed and initialized)